app/models/unlocalized_words.php

```diff
@@ -0,0 +1,104 @@
+<?php
+namespace Transvision;
+
+// Build arrays for the search form, ignore mozilla_org
+$channel_selector = Utils::getHtmlSelectOptions(
+    $repos_nice_names,
+    $repo,
+    true
+);
+$target_locales_list = Utils::getHtmlSelectOptions(
+    Project::getRepositoryLocales($repo),
+    $locale
+);
+
+$ref_locale = Project::getReferenceLocale($repo);
+
+/*
+    pspell helps getting rid of false positive results by keeping only valid
+    English words. The downside is that it’s filtering out 'jargon' words that
+    can be used in devtools or Mozilla-specific words.
+
+    This requires the following packages to be installed:
+    sudo apt-get install libpspell-dev php5-pspell aspell-en
```

Comment:
Looks like what I've found is just a wrapper for the package https://github.com/prodeveloper/pspell/blob/master/src/Chencha/Pspell/Validators/CheckPspellIsInstalled.php#L13

However, pspell should be available for Windows http://code.stephenmorley.org/php/enabling-pspell-in-xampp-on-windows/